app/bundles/DashboardBundle/Resources/views/Dashboard/recentactivity.html.twig

```diff
@@ -9,25 +11,22 @@
                         </span>
                     </div>
                     <div class="media-body">
-                        {% if log.userId is defined and log.userId is not empty %}
+                        {% if log.userName == system_user or (log.userId is not defined or log.userId is empty) %}
```

Comment:
We can enhance the original if statement to `{% if log.userId is defined and log.userId != 0 %}` instead of verifying the username against the system user.
Once the activity is recognized as a system (anonymous), it records this in the audit log table with the `userId` set to 0.